web/client/components/widgets/enhancers/dependenciesToZoomTo.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { useEffect, memo } from 'react';
+
+const dependenciesToZoomTo = (WrappedComponent)=>{
+    function DependenciesToZoomTo(props) {
+        let tblWidgetWithExtentObj = props.widgets.find(i=>i?.dependencies?.extentObj);
```

Comment:
with this line: what if you have more table widgets connected to a map widget

Reply:
I will check this case

Reply:
Here I clear extentObj after each zoom process. So extentObj will be available with just on table that user clicks on.